groups/wg-k8s-infra/groups.yaml

```diff
@@ -111,6 +111,7 @@ groups:
       - spiffxp@gmail.com
       - spiffxp@google.com
       - thockin@google.com
+      - jdagostino2@gmail.com
```

Comment:
@jimdaga 👋🏾  can you keep this list ordered alphabetically ?

Reply:
@ameukam  I have corrected this (also fixed one existing name that was out of order).